modules/schema-util/src/main/scala/edu/gemini/schema/util/SchemaStitcher.scala

```diff
@@ -0,0 +1,177 @@
+// Copyright (c) 2016-2023 Association of Universities for Research in Astronomy, Inc. (AURA)
+// For license information see LICENSE or https://opensource.org/licenses/BSD-3-Clause
+
+package edu.gemini.schema.util
+
+import cats.data.{Ior, NonEmptySet}
+import cats.effect.Sync
+import cats.kernel.Order
+import cats.parse.Parser.*
+import cats.parse.Rfc5234.{alpha, digit, wsp}
+import cats.parse.{Parser, Parser0}
+import cats.syntax.all.*
+import cats.{Applicative, Eq, Monad}
+import edu.gemini.grackle.*
+import eu.timepit.refined.types.string.NonEmptyString
+
+import scala.io.Source
+import scala.tools.nsc.io.Path
+
+// SchemaStitcher can build a grackle Schema from schema files that contains import statements of the form:
+// #import <type list> from <schema file name>
```

Comment:
Is this syntax well known or would it be a custom style?
Could the `<schema file name>` be a url?

Reply:
For the syntax I took inspiration on graphql-import, a JavaScript package now merged into GraphQl Tools.
About the second question, notice that the resolution of the schema source from the `<schema file name>` is delegated to an instance of `SourceResolver` provided by the user. So you could use URLs if you provide an implementation of `SourceResolver` that could fetch the schema from the URL.